tutorials/data/combining with syncer example project/group/02334ffa-53b9-44bb-970f-c62a916ebea8.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "tutorials/data/Combining with syncer example project/Group",
```

Comment:
Should have more meaningful name.
Review the port and node names.
Review the data types.